File: even-odd/evenodd.java
//created by- Siddharth Saurabh (https://github.com/siddhartthecoder)
import java.util.*;
class evenodd {
 public static void main(String args[]) throws Exception {
  Scanner sc = new Scanner(System.in);
  System.out.println("Enter your number:");
  int n = sc.nextInt();
  if(n%2==0)
    System.out.println(n+" is even");
  else
    System.out.println(n+" is odd");
 }
}

File: vmsantos/even_odd.c
#include <stdio.h>

void main(){
	printf("\n*****Check wether a number is even or odd*****\n\n\n");
	
	int a;
	
	printf("Enter a number:\n");
	scanf("%d", &a);

	if (a % 2 == 0)
		printf("--->The number you entered is even\n");
	else
		printf("--->The number you entered is odd\n");
}
